### PR TITLE
feat: reference service 0.2.0

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: docs/migrating_to_16.md [hunk 1]
```diff
@@ -11,16 +11,51 @@ Key points:
    * If needed, the default service specific volume directories defined in [bento.env](../etc/bento.env) can be overriden in your 
      `local.env`.
 * The WES schema has been updated, so the run database must be cleared.
+* The reference service has been updated, so a SQL migration must be run.
```
Comment (gsfk) [inline docs/migrating_to_16.md:14]: "an SQL" rather than "a SQL"

... sorry

**Review by gsfk — COMMENTED** [review]
Looks good, only one annoying inline comment. 

My local reference service was already on recent master, so I'm not able to test the migration currently.

**Review by v-rocheleau — CHANGES_REQUESTED** [review]

File: docs/migrating_to_16.md [hunk 2]
```diff
@@ -11,16 +11,51 @@ Key points:
...
 
 
-## 1. Pre-update data config
+## 1. Run reference service pre-migration step
+
+```bash
+./bentoctl.bash shell reference-db
+PGPASSWORD="${POSTGRES_PASSWORD}" psql --user "${POSTGRES_USER}" reference
```
[the remark this answers is not in the document]
Comment (davidlougheed) [reply docs/migrating_to_16.md:21]: when it's on the same line it only sets it for the command, not for the shell generally, but I agree that in this case it'll be clearer and theres no harm in doing so

File: docs/migrating_to_16.md (hunk 1 above)
Comment (davidlougheed) [reply docs/migrating_to_16.md:14]: "sequel" or "ess-queue-ell"... that is a question

Comment (gsfk) [reply docs/migrating_to_16.md]: ah, I guess I use the geriatric version

**Review by gsfk — APPROVED** [review]

File: etc/bento.env [hunk 3]
```diff
@@ -97,7 +97,7 @@ BENTO_AUTHZ_DB_MEM_LIM=1G
 # Web
 BENTO_WEB_CUSTOM_HEADER=
 BENTOV2_WEB_IMAGE=ghcr.io/bento-platform/bento_web
-BENTOV2_WEB_VERSION=edge
+BENTOV2_WEB_VERSION=pr-405
```
Comment (v-rocheleau) [inline etc/bento.env:100]: Put back to "edge" once the web PR 405 is merged

**Review by v-rocheleau — APPROVED** [review]
Lgtm

**Pull request merged**